### DISCUSSION TITLE
Trace data capture time can be set

### DISCUSSION THREAD

**Discussion by mitht** [apache/skywalking#11581] [Q&A] [answered]
I added the application to skywalking-agent and called the background interface through postman; how long will it take for the trace data to be displayed on the page? Can this time be set?

![image](https://github.com/apache/skywalking/assets/29425860/15d40e0f-daf1-4b84-a90e-8598aec587e4)

**Comment by wu-sheng** [comment, marked as answer]
It would be shown in seconds, if not, your agent isn't installed correctly.

**Reply by mitht** [reply]
Can this time be set in the configuration file?
